crates/sol-macro/src/expand/contract.rs

```diff
@@ -136,14 +162,33 @@ impl CallLikeExpander {
             variants: errors.iter().map(|error| error.name.0.clone()).collect(),
             min_data_len: errors
                 .iter()
-                .map(|error| r#type::params_base_data_size(cx, &error.parameters))
+                .map(|error| ty::params_base_data_size(cx, &error.parameters))
                 .min()
                 .unwrap(),
             trait_: Ident::new("SolError", Span::call_site()),
             data: CallLikeExpanderData::Error { selectors },
         }
     }
 
+    /*
```

Comment:
what's the status of this code?

Reply:
refactored to expand the enum but no SolInterface trait impl